apps/oxlint/src-js/plugins/lint.ts

```diff
@@ -142,6 +148,9 @@ export function lintFileImpl(
   // Pass settings JSON to context module
   setSettingsForFile(settingsJSON);
 
+  // Pass globals JSON to context module
```

Comment:
```suggestion
  // Pass globals JSON to globals module
```

The comment above for settings was also inaccurate. Fixed that in #16559.